lib/sail/value_cast.rb

```diff
@@ -12,6 +14,10 @@ def range_get(value)
       value.to_i
     end
 
+    def date_get(value)
+      Time.parse(value).strftime("%d-%m-%Y")
```

Comment:
I believe we should use DateTime instead of Time. Furthermore, I think the formatting (strftime) should be done by the client using the interface.

We can return the result of DateTime.parse and return the date object. This will give more options to people using the gem.